app/js/index.js

```diff
@@ -106,12 +115,11 @@ function init() {
       return +d.properties[field];
     });
 
-  var topoURL = DATA_DIRECTORY + "us-states.topojson";
-  //var topoURL = DATA_DIRECTORY + "CAcounty.topojson";
-  d3.json(topoURL, function(topology) {
+  
+  d3.json(DEFAULT_TOPO, function(topology) {
```

Comment:
Eventually we'll need some way to pass in the topo that the user chose from our list right?